src/modules/blog/entities/blog.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { Entity, Column, ManyToOne, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { User } from '../../user/entities/user.entity';
+import { BlogComment } from './blog-comment.entity';
+import { BlogCategory } from './blog-category.entity';
+
+@Entity()
+export class Blog extends AbstractBaseEntity {
+  @PrimaryGeneratedColumn('uuid')
```

Comment:
remove this since you extended the AbstractBaseEntity. It has been provided in the extended class

Reply:
Removed